src/crawlers/lotteria.py

```
@@ -0,0 +1,251 @@
+            logger.error(f"Error during crawling {self.brand_name}: {e}")
+        except json.JSONDecodeError as e:
+            logger.error(f"Error decoding JSON from pList: {e}")
+        except Exception as e:
+            logger.error(f"An unexpected error occurred: {e}")
+        finally:
+            # 드라이버 정리
+            if driver:
+                try:
+                    driver.quit()
+                    logger.info("Driver closed successfully")
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)